### PR TITLE
Set up my first mobile

### PR DESCRIPTION
**Pull request by AlexisJW**
This pull request has been created to replace every child element with a section tag with generic and change Let's connect to a link.

### CODE REVIEW

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @AlexisJW, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+  <body>
+    <div class="containers">
+      <header class="navbar-container">
+        <a class="logo" href="#"> AlexisJW </a>
+        <nav id="navbar">
+          <div class="About"></div>
+          <div class="My works"></div>
+          <div class="Contact"></div>
+        </nav>
+      </header>
```
Comment (Gambit142) [inline index.html:20]: - [ OPTIONAL] The main container of the **_toolbar/navbar_** section should be the header tag, not the div tag. This makes it more semantic.

```HTML
 <header class="containers">
      <div class="navbar-container">
```

File: index.html [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+	  
+     <div class="section-1">  
+      <div class="image-background">
+         <div class="headline">
+            <section>
```
Comment (Gambit142) [inline index.html:25]: - [ OPTIONAL] The main container of the **_headline_** section should be the section tag, not the div tag. This makes it more semantic.

```HTML
 <section class="headline">
            <div>
```

**Review by Gambit142 — APPROVED** [review]
Hi @AlexisJW 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- All linters checks passed ✔️ 
- Implemented all the changes requested by the previous reviewer ✔️ 

### Optional Suggestion

Kindly consider the suggestions marked **[OPTIONAL]** in the comments under the review. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**